data_contribution_2_data.md

```diff
@@ -0,0 +1,907 @@
+
+**Within an [Arcus Scientific Lab](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/main/arcus_orientation.md#1)**
+
+- External or public datasets not supplied by Research IS or your lab, such as census data, will be available in this directory.
+
+**Ref-data** data is different depending on the type of research. Please select what below if need need more information about omics data for this directory:
+
+- [ ] omics data
+- [ ] clinical data
+<script output="data_type_ref">"@input"</script>
+
+<script modify="false">
+try {
+  let data_type = @input(`data_type_ref`)
+
+  if(data_type[0]) {
+    send.liascript(`
+    ## Omics Data
+
+- As cram files are a compressed format, some information is needed as a seperate file. If possible we collect a fasta file for the reference genome used. The fasta file describes offsets for each contig, to compute exactly where to find a particular reference base at specific genomic coordinates. Each fasta file requires an index file as fasta.fai
+- bed files are a text file format used to store genomic regions as coordinates and associated annotations, see [documentation](https://samtools.github.io/hts-specs/BEDv1.pdf) for more information on the format. If available, we collect this file as a .bed extension.
+
+
+    `)
+  } else send.clear()
+} catch(e) { }
+</script>
+
+
+<script modify="false">
+try {
+  let data_type = @input(`data_type_ref`)
+
+  if(data_type[1]) {
+    send.liascript(`
+    ## Clinical Data
+
+Arcus Archives does not collect clinical reference data sets but there are ways to tap into the power of these datasets within Arcus.
+- The Arcus Applied Data Sciences team offers [GIS Data](https://chop.alationcloud.com/domain/30/) of many high value publicly available regularly refreshed Geographical Information System Datasets.
+- There is also an option via [Jira Service Desk](https://jira.arcus.chop.edu:8443/servicedesk/customer/portal/6/create/355?q=upload&q_time=1696276238126) to upload reference data directly to your lab following disclosure and approval.
+
+    `)
+  } else send.clear()
+} catch(e) { }
+</script>
+
+
+## manifests/
+
+Manifests are an inventory of all data in the collection, and provide a mapping between research data in the data folders, and participant information. The manifests also create a mapping between data and associated pipeline and technical information about workflows. There are three main manifests that are mandatory for every archival collection:
+
+- file_manifest.csv
+- participant_manifest.csv
+- partcipant-crosswalk.txt
+
+Additional manifests are only required if needed for the data or collection type. These files are detailed in the next sections. The graphic below illustrates the linking between the files:
+
+![ID Crosswalks between manifests](media/project_template/manifests_linkages.png)
```

Comment:
Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases. I leave it to you to decide if the effort of re-doing is worth it.